routers/postRouter.js

```diff
@@ -0,0 +1,39 @@
+const express = require('express');
+const router = express.Router();
+const db = require('../server/helpers/postHelpers');
+
```

Comment:
Very solid code Michael 🔥 Any reason you decided to use the name `posts` instead of `notes`? Either one works but I was just curious 👍